README.md

```diff
@@ -34,6 +34,11 @@ Examples:
 
 (`host/managed` is for Foreman 1.2+, change to just `host` for Foreman 1.1)
 
+Plugins need to add the plugin name before the model name. For example:
+
+    ~/foreman/config/hooks/katello/product/before_create/01_integrate_catalog.sh
+    ~/foreman/config/hooks/foreman_tasks/task/dynflow_task/after_create/90_record_action.sh
```

Comment:
With the new changes, this example does work. TBH..I am not sure how the STI _should_ play here. TBH.. I would be willing to go with it as is since I would expect whoever uses hooks to know the obejects somewhat.